WhaleSpotting/react-app/src/components/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import "../styles/Card.scss";
+import React, { useState } from "react";
+import SightingApiModel from "../apiModels/SightingApiModel";
+
+interface CardProps {
+    sighting: SightingApiModel;
+}
+
+export default function Card({sighting}: CardProps) : JSX.Element {
+
+    const [closeCard, setCardState] = useState(true);
+    
+    //TODO - authorise admin to display userIds
+    const admin = true;
+    
+    return (
+        <div className="card-component" data-testid="card-component">
+            <div className={!sighting.confirmed ? "pending" : "confirmed"}
+                data-testid={!sighting.confirmed ? "pending" : "confirmed"}> PENDING </div>
```

Comment:
instead of doing this, we could just make it so that we only show this div if its pending
```suggestion
            {!sighting.confirmed && <div className="pending" data-testid="pending"> PENDING </div>}
```
That way we can remove the confirmed class

Reply:
cool! done!